connectors/src/connectors/microsoft/index.ts

```diff
@@ -32,6 +34,7 @@ import {
   launchMicrosoftFullSyncWorkflow,
   launchMicrosoftIncrementalSyncWorkflow,
 } from "@connectors/connectors/microsoft/temporal/client";
+import { MicrosoftNodeModel } from "@connectors/lib/models/microsoft";
```

Comment:
hmm we shouldn't have sequelize/models imported in here
the logic should move to resource or elsewhere; will have a look below to see where